docs/_snippets/features/word-count-update.js

```diff
@@ -28,39 +28,39 @@ ClassicEditor
 		},
 		table: {
 			contentToolbar: [ 'tableColumn', 'tableRow', 'mergeTableCells' ]
-		}
-	} )
-	.then( editor => {
-		const wordCountPlugin = editor.plugins.get( 'WordCount' );
-
-		const progressBar = document.querySelector( '.customized-count progress' );
-		const colorBox = document.querySelector( '.customized-count__color-box' );
+		},
+		wordCount: {
```

Comment:
The code has diverged from the description in the docs.

Since the paragraph is about `update` event the code should work with the same event and be 1:1 (minus the obvious bloat) with an example. 

Side note - could you add also information about this to that docs in the update event section? I mean that the same can be achieved by using `onUpdate` configuration option.

Reply:
I was confused by what this PR does and left this comment: https://github.com/ckeditor/ckeditor5-word-count/issues/18#issuecomment-518252835.

Additional note: The documentation should be reviewed to show `onUpdate` as the "go-to solution" and only mention the event as a side note, explaining the fact that it cannot be used for this scenario.